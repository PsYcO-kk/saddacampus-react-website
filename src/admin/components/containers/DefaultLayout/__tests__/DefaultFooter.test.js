import React from 'react';
import { shallow } from 'enzyme';
import DefaultFooter from '../DefaultFooter';

test('should render DefaultFooter component without crashing', () => {
	const wrapper = shallow(<DefaultFooter />);
	expect(wrapper).toMatchSnapshot();
});
