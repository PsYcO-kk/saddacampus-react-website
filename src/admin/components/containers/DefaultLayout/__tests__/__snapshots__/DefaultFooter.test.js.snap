// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DefaultFooter component without crashing 1`] = `
<React.Fragment>
  <span>
    <a
      href="#"
    >
      Saddacampus
    </a>
     ©
  </span>
  <span
    className="ml-auto"
  >
    Powered by 
    <a
      href="https://coreui.io/react"
    >
      CoreUI for React
    </a>
  </span>
</React.Fragment>
`;

exports[`should render DefaultFooter correctly 1`] = `
<React.Fragment>
  <span>
    <a
      href="#"
    >
      Saddacampus
    </a>
     ©
  </span>
  <span
    className="ml-auto"
  >
    Powered by 
    <a
      href="https://coreui.io/react"
    >
      CoreUI for React
    </a>
  </span>
</React.Fragment>
`;
